packages/settingeditor-extension/src/index.ts

```diff
@@ -52,6 +52,11 @@ import {
   ImportSettingsWidget
 } from './importSettingsWidget';
 
+const HARDCODED_TO_SKIP = [
+  '@jupyterlab/application-extension:context-menu',
+  '@jupyterlab/mainmenu-extension:plugin'
+];
```

Comment:
Personally I would make this a default rather than hard-coded.

Reply:
Sure, I think the idea was to keep the existing behavior.

Maybe we could check why these two plugins were specified in the list to see if there would be side-effects of having them as defaults in the settings.

Reply:
I think this was because originally all plugins were rendered together and it was slowing opening the settings panel down. This is no longer the case, but if someone searched for "menu" they would still get a slowdown.